src/cmake/dependencies.cmake

```diff
@@ -127,6 +127,44 @@ if(BUILD_PYTHON_BINDINGS)
   endif()
 endif()
 
+if(BUILD_WITH_BULLET AND NOT USE_SYSTEM_BULLET)
+  # The below block except for the visiblity patch verbatim copied from
+  # https://doc.magnum.graphics/magnum/namespaceMagnum_1_1BulletIntegration.html
+
+  # Disable Bullet tests and demos
+  set(BUILD_UNIT_TESTS OFF CACHE BOOL "" FORCE)
+  set(BUILD_BULLET2_DEMOS OFF CACHE BOOL "" FORCE)
+  set(BUILD_CPU_DEMOS OFF CACHE BOOL "" FORCE)
+  set(BUILD_OPENGL3_DEMOS OFF CACHE BOOL "" FORCE)
+  # While not needed for Magnum, you might actually want some of those
+  set(BUILD_ENET OFF CACHE BOOL "" FORCE)
+  set(BUILD_CLSOCKET OFF CACHE BOOL "" FORCE)
+  set(BUILD_EXTRAS OFF CACHE BOOL "" FORCE)
+  set(BUILD_BULLET3 OFF CACHE BOOL "" FORCE)
+  # This is needed in case BUILD_EXTRAS is enabled, as you'd get a CMake syntax
+  # error otherwise
+  set(PKGCONFIG_INSTALL_PREFIX "lib${LIB_SUFFIX}/pkgconfig/")
+  # Bullet's buildsystem doesn't correctly express dependencies between static
+  # libs, causing linker errors on Magnum side. If you have CMake 3.13, the
+  # Find module is able to correct that on its own, otherwise you need to
+  # enable BUILD_SHARED_LIBS to build as shared.
+  if(NOT CORRADE_TARGET_EMSCRIPTEN)
```

Comment:
Can you add a version check here and build static if >= 3.13?  I'd like to statically link this for conda as that'll reduce the number of things we need to worry about rpath. The conda-build is just grabbing latest cmake, so no need to worry about versions there.